### PR TITLE
Added networking_downloading sample app

### PR DESCRIPTION
**Pull request by ggrosso**
Update to the final version of the networking app called networking_downloading

### CODE REVIEW

File: All_Samples.json [hunk 1]
```diff
@@ -1499,6 +1499,13 @@
      "repourl": "http://github.com/blackberry/BB10-WebWorks-Community-Samples",
      "tags": [ "html5", "webworks", "bb10", "nfc", "invocation" ]
  },
+ "Networking_v1_0": {
```
Comment (timwindsor) [inline All_Samples.json]: The name doesn't match the one in the repo (networking_downloading)

File: All_Samples.json [hunk 2]
```diff
@@ -1499,6 +1499,13 @@
...
+     "desc": "A networking sample that shows how to retrieve an XML file from a remote server while determining network connectivity and the connection interface being used.",
+     "url": "https://github.com/blackberry/Cascades-Samples/tree/master/networking_v1_0",
```
Comment (timwindsor) [inline All_Samples.json]: This location is going to be wrong with the name that was used in the end:
https://github.com/blackberry/Cascades-Samples/tree/master/networking_downloading

Comment (ggrosso) [inline All_Samples.json]: This link does go to the sample. Not sure why you think it is wrong. Please let me know.

Comment (ggrosso) [inline All_Samples.json]: Hi Tim,
I updated the name of the sample in that file to networking_downloading, but I am not seeing the update in your message.
Please note that my changes may not have been merged, and that may explain why I cannot see them in the repo. The sample is called networking_downloading, I have commented on this several times in the repo. Please let me know what needs to be done to make the necessary correction so we can avoid this error.

Thanks,
George Grosso
[BlackBerry]

**Comment by timwindsor** [comment]
Yes, the name is changed in the sample repo, and now in the readme for the sample as well.

The name and url in this pull request (for the catalog) is still referring to the old networking_v1_0 name, though. My comment was to remind you of the url that it should be using. If you click on the url that is in your current pull request it will go to a 404 page.

**Comment by ggrosso** [comment]
Are you referring to the two old requests shown above?

https://github.com/ggrosso/Catalogs/commit/482079ad0adad7510bf49a9ff75c698ae2134680
https://github.com/ggrosso/Catalogs/commit/3953285688d6a0600c8503336540f42360ed8d19

Those two requests are 4 months old, but I don't know how to remove them. Can you remove them for me?
Also, is there anything else that I can do to make sure that the old networking_v1_0 name is not causing a problem. Please let me know.

**Comment by ggrosso** [comment]
@timwindsor 

After some careful investigation, I think that I may have been on the wrong branch. I have checked out the gh-pages branch, made the name correction, and now I think that all is good to go. Please verify that this is correct for me, and let me know if you can merge the change.

**Comment by timwindsor** [comment]
Yes, it looks like your latest commits have cleaned up the changes to match with what I was expecting to see.

**Pull request merged**